operational_analysis/methods/electrical_losses.py

```diff
@@ -158,7 +160,51 @@ def calculate_electrical_losses(self):
         if self._monthly_meter:
             scada_monthly = self._scada_daily.resample('MS')['corrected_energy'].sum().to_frame()
             scada_monthly.columns = ['turbine_energy_kwh']
+
+            # Determine availability for each month represented
+            scada_monthly['count'] = self._scada_sum.resample('MS')['count'].sum()
+            scada_monthly['expected_count_monthly'] = 1 #initialized values for column
+
+            #months with 31 days
+            month_list = ((scada_monthly.index.month ==1) | \
+            (scada_monthly.index.month == 3) | \
+            (scada_monthly.index.month ==5) | \
+            (scada_monthly.index.month ==7) | \
+            (scada_monthly.index.month ==8) | \
+            (scada_monthly.index.month ==10) | \
+            (scada_monthly.index.month ==12))
+            scada_monthly.loc[month_list, 'expected_count_monthly'] = \
+            self._month_hours[0]* self._plant._num_turbines /self._time_conversion[self._plant._scada_freq]
+
+            #February 
+            month_list = (scada_monthly.index.month ==2)
+            year_list = ((scada_monthly.index.year == 2000) | \
+            (scada_monthly.index.year == 2004) | \
+            (scada_monthly.index.year == 2008) | \
+            (scada_monthly.index.year == 2012) | \
+            (scada_monthly.index.year == 2016) | \
+            (scada_monthly.index.year == 2020))
+
+            #Non Leap year
+            scada_monthly.loc[(month_list & year_list), 'expected_count_monthly'] = \
+            self._month_hours[1]* self._plant._num_turbines /self._time_conversion[self._plant._scada_freq]
+            
+            #Leap year
+            scada_monthly.loc[(month_list & (~year_list)), 'expected_count_monthly'] = \
+            self._month_hours[2]* self._plant._num_turbines /self._time_conversion[self._plant._scada_freq]
+
+            #months with 30 days
+            month_list = ((scada_monthly.index.month ==4) | \
+            (scada_monthly.index.month ==6) | \
+            (scada_monthly.index.month ==9) | \
+            (scada_monthly.index.month ==11))
+            scada_monthly.loc[month_list, 'expected_count_monthly'] = \
+            self._month_hours[3]* self._plant._num_turbines / self._time_conversion[self._plant._scada_freq]
+
+            scada_monthly['perc'] = scada_monthly['count']/scada_monthly['expected_count_monthly']
             
+            # Filter out months in which there was less than 95% of total running (all turbines at all timesteps)
+            scada_monthly = scada_monthly.loc[scada_monthly['perc']>= .95, :]
```

Comment:
Should this be parametrized? Is 0.95 a standard? If so, please reference the standard. If not, let's make it a parameter.